geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementResult.java

```diff
@@ -90,9 +89,35 @@ public ClusterManagementResult(ClusterManagementResult copyFrom) {
   /**
    * for internal use only
    */
-  public void setStatus(StatusCode code, String message) {
-    this.statusCode = code;
-    this.statusMessage = message;
+  public void setStatus(StatusCode statusCode, String message) {
+    this.statusCode = statusCode;
+    this.statusMessage = proper(message);
+  }
+
+  /**
+   * Capitalized the first letter and adds a period, if needed.
+   * This helps the CMS API give a consistent format to error messages, even when they sometimes
+   * come from parts of the system beyond our control.
+   */
+  private static String proper(String message) {
```

Comment:
Can we give this a more descriptive name? Something like `formatErrorMessage`?

Reply:
done